### ISSUE TITLE
Feature request - Wrapper for Fourier Transform

### ISSUE THREAD

**Issue by SebastienJoly** [neffint/neffint#12]
So far the functions implemented are Inverse Fourier Transform, converting frequency domain signal into time domain. While it is well suited to convert impedance to wake function it would be useful to do the inverse, namely converting time domain signal into frequency domain.
Thanks to Fourier Transform properties it should be fairly easy knowing an Inverse Fourier Transform of f(x) is nothing more than a Fourier Transform of the complex conjuguate of f(x). Please don't quote me on this there may be a normalization constant to take into account depending on the Fourier Transform convention used.

Basically the wrapper function would be equivalent to this :
transform_arr = nft.fourier_integral_fixed_sampling(
    times=frequencies, #switch times and frequencies
    frequencies=times, #switch times and frequencies
    func_values=1j*func_values, #func_values is the time domain signal
    interpolation="pchip"
)
